plugin/http/propagation/propagation_test.go

```diff
@@ -0,0 +1,109 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package propagation
+
+import (
+	"net/http"
+	"reflect"
+	"testing"
+
+	"go.opencensus.io/trace"
+)
+
+func TestHTTPFormat_FromRequest(t *testing.T) {
+	tests := []struct {
+		name   string
+		header string
+		wantSc trace.SpanContext
+		wantOk bool
+	}{
+		{
```

Comment:
Let's also add an all zero ie [16]byte traceID and an all zero spanID too in our tests.

Reply:
Done